### ISSUE TITLE
Estimation on the duration of AI training

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#103]
Siemens asked us for a estimation on the number of hours required to train the AI throughout the whole project duration.

## Acceptance criteria
- [ ] research on the topic was conducted
- [ ] at least 2 developer agreed on the respective number 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
